### PR TITLE
Fix wget integration in Python 3

### LINKED ISSUE #82: Fix wget integration [Python 3]
For some reason the wget integration (#68) worked just fine when I tested it. I downloaded new files, tried to see if you can continue an interrupted download and it did not give errors on files that have finished downloading. But it looks like there are several issues now; probably I had a different environment.

EDIT: Looks like this is an issue only in Python 3

### PR DESCRIPTION
**Pull request by SchnWalter**
Fixes #82

### CODE REVIEW

File: mia/helpers/utils.py [hunk 1]
```diff
@@ -327,40 +327,51 @@ def urlretrieve(url, install_filepath, cache_path=None):
...
-    if cache_enabled:
-        filename = url.split('/')[-1]
-        download_filepath = os.path.join(cache_path, filename)
-    else:
+    if cache_path is None:
+        cache_enabled = False
         download_filepath = install_filepath
+    else:
+        cache_enabled = True
+        wget_arguments.append('--continue')
```
Comment (patcon) [inline mia/helpers/utils.py]: Why would we only continue a partial download when we're caching?

Comment (SchnWalter) [inline mia/helpers/utils.py]: A lazy, quick and dirty fix for the outdated index.xml issue.

Comment (patcon) [inline mia/helpers/utils.py]: If this is our way around the index.xml issue, I'm a-ok with this being a temporary resolution, but just lemme know so I can open another issue for doing it a little more intentionally and reverting this later :)

**Comment by patcon** [comment]
Good to go aside from that outstanding inline comment

**Comment by SchnWalter** [comment]
OK then, I will merge this in, because my last attempt to fix this actually broke PY2 also, so without these changes downloading is not working at all :D

**Comment by patcon** [comment]
:+1: woo!

**Pull request merged**